src/wwwroot/template-editor/adapterService.js

```diff
@@ -1,6 +1,9 @@
 (function () {
 
   'use strict';
+  // Add dopplerRelay module to take environment configuration constant
+  angular
+    .module("mseditor").requires.push('dopplerRelay');
```

Comment:
How does it work?

Reply:
This adds a new module to the current scope of another one in runtime. We use this to prevent override the initial bootstrapped modules from mseditor.

We can't use the next to define it due to this override all the previous bootstrapped modules in mseditor application:
```js
angular.module("mseditor", [dopplerRelay])
```

Reply:
Thanks for the clarification!